DevelopGitHub/GitHubExtension/GitHubExtension.Templates.WebApi/Controllers/TemplatesController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Web.Http;
+using GitHubExtension.Templates.WebApi.Constants;
+using GitHubExtension.Templates.WebApi.Services;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Templates.WebApi.Controllers
+{
+    [RoutePrefix(RouteTemplatesConstants.GetGitHubTemplatesRoute)]
+    public class TemplatesController : BaseApiController
+    {
+        #region private fields
+
+        private readonly ITemplateService _templateService;
+        private const string CurrentProjectName = "CurrentProjectName";
+        private const string PathToPullRequestTemplate = ".github/PULL_REQUEST_TEMPLATE.md";
+        private const string PathToIssueTemplate = ".github/ISSUE_TEMPLATE.md";
+        #endregion
+
+        public TemplatesController(ITemplateService templateService)
```

Comment:
Currently we are using CQRS approach...Please use correct naming for ITemplateService or consult with colleagues. They already use this approach and naming.